builders/xcm/xcm-sdk/reference.md

```diff
@@ -0,0 +1,65 @@
+
+The SDK provides the following core methods, which will be covered in this guide:
+
+|                                           Method                                            |                                      Description                                      |
+|:-------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------------------------:|
+|           [`init()`](/builders/xcm/xcm-sdk/xcm-sdk/#initializing){target=_blank}            |    Initializes the XCM SDK. **Must be called first before any other SDK methods**     |
+|            [`deposit()`](/builders/xcm/xcm-sdk/xcm-sdk/#deposit){target=_blank}             |         Initiates a deposit to transfer assets from another chain to Moonbeam         |
+|           [`withdraw()`](/builders/xcm/xcm-sdk/xcm-sdk/#withdraw){target=_blank}            |        Initiates a withdraw to transfer assets from Moonbeam to another chain         |
+| [`subscribeToAssetsBalanceInfo()`](/builders/xcm/xcm-sdk/xcm-sdk/#subscribe){target=_blank} |   Listens for balance changes for a given account for each of the supported assets    |
+|     [`isXcmSdkDeposit()`](/builders/xcm/xcm-sdk/xcm-sdk/#deposit-check){target=_blank}      | Returns a boolean indicating whether the given transfer data is for a deposit or not  |
+|    [`isXcmSdkWithdraw()`](/builders/xcm/xcm-sdk/xcm-sdk/#withdraw-check){target=_blank}     | Returns a boolean indicating whether the given transfer data is for a withdraw or not |
+|          [`toDecimals()`](/builders/xcm/xcm-sdk/xcm-sdk/#decimals){target=_blank}           |                       Returns a given balance in decimal format                       |
+
+## Deposit Methods {: #deposit-methods }
+
+When building the transfer data needed for a deposit, you'll use multiple methods to build the underlying XCM message and send it:
+
+|                                   Method                                    |                                                                                     Description                                                                                      |
+|:---------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+|    [`deposit()`](/builders/xcm/xcm-sdk/xcm-sdk/#deposit){target=_blank}     |                             Initiates a deposit to transfer assets from another chain to Moonbeam. **Must be called first before you can call `from()`**                             |
+|       [`from()`](/builders/xcm/xcm-sdk/xcm-sdk/#from){target=_blank}        |                                  Sets the source chain where the deposit will originate from. **Must be called first before you can call `get()`**                                   |
```

Comment:
Would it make sense to say that the `from` and others are inherited or return functions after calling `deposit`?